src/main/java/data/Database.java

```diff
@@ -0,0 +1,79 @@
+package data;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.ArrayList;
+
+/* This class creates an instance of a database that a file can be written to and read from.
+Assumptions: for each runID, only 1 of each datatype can be associated with it. */
+public class Database {
+  /* Place where files are stored.*/
+  private String directory;
+
+  /* List of all files stored in Database.*/
+  private ArrayList<String> files;
+
+  /* Creates instance of a database with this name. */
+  public Database(String name) {
+    this.directory = name;
+    files = new ArrayList<String>();
```

Comment:
I think it is better to list the files under the directory than init with empty list. Since if you restart your db service, you'll lost all existing files.